e2e/newenemy/newenemy_test.go

```diff
@@ -344,8 +345,8 @@ func checkDataNoNewEnemy(ctx context.Context, t testing.TB, schemaData []SchemaD
 		r1leader, r2leader := getLeaderNode(ctx, crdb[1].Conn(), exclude.Tuple), getLeaderNode(ctx, crdb[1].Conn(), direct.Tuple)
 		ns1Leader := getLeaderNodeForNamespace(ctx, crdb[1].Conn(), exclude.Tuple.ObjectAndRelation.Namespace)
 		ns2Leader := getLeaderNodeForNamespace(ctx, crdb[1].Conn(), exclude.Tuple.User.GetUserset().Namespace)
-		z1, _ := zookie.DecodeRevision(r1.GetRevision())
-		z2, _ := zookie.DecodeRevision(r2.GetRevision())
+		z1, _ := zookie.DecodeRevision(core.CoreZookie(r1.GetRevision()))
```

Comment:
Might be worth naming these slightly differently, like `V0ToCoreZookie`

Reply:
I actually started with the XToY naming but it got hard to read so I landed on just the desired type as the name.

Reply:
Then maybe just `ToCoreZookie`? The current names are a bit ambiguous to me